functions/src/world.ts

```diff
@@ -146,3 +146,76 @@ export const deleteWorld = functions.https.onCall(async (data, context) => {
 
   admin.firestore().collection("worlds").doc(worldId).delete();
 });
+
+export const addWorldAdmin = functions.https.onCall(async (data, context) => {
+  assertValidAuth(context);
+
+  const worldId = data.worldId;
+  const userId = data.userId;
+
+  await throwErrorIfNotWorldOwner({
+    worldId,
+    userId,
+  });
+
+  const worldRef = await admin
+    .firestore()
+    .collection("worlds")
+    .doc(worldId)
+    .get();
+
+  const worldData = worldRef.data();
+
+  if (!worldData) {
+    return;
+  }
+
+  const owners = worldData.owners as string[];
```

Comment:
[NOTE] might be OOS:

I think it would be more helpful to have some type guard here that uses `Array.isArray()` so we're not just blindly assuming it's an array. Might be useful to just use `Array.isArray()` without spending time on creating type guards like 

```ts
if( Array.isArray(owners) && owners.includes(userId) )
```

The `return` that follows, however, to avoid silent errors, might be useful to check if the simplest log like `console.error()` (or `warn` or `log`) is good enough to have some info of the problem in the Firestore functions console.

Reply:
The above note is valid for other similar instances, if deemed OOS, we should think of this kind of improvement for the future

Reply:
The proposal is valid and better than nothing. However we need to think of a more global way to type these database responses.